README.rdoc

```diff
@@ -0,0 +1,28 @@
+== README
+
```

Comment:
It's always useful to change this README to something that describes your project. (I'm sure this wasn't a requirement for this project, but it's something I wish someone had mentioned to me earlier on.) Check out some other projects to get a sense of what might go here. Makes everything look more profesh.